masterlist.yaml

```diff
@@ -4004,6 +4025,7 @@ plugins:
         name: 'Legacy of the Dragonborn Patches SE on Nexus Mods'
     tag:
       - Actors.AIPackages
+      - Delev
```

Comment:
This delev tag isn't needed, the lvl list record in this patch merges the level list changes from LOTD & Cutting Room floor. 
Delev tag is used when an entry is removed. 
Just remember at xEdit startup you should only tick the plug-in you want to check for bash tags.
As checking with other plug-ins can give you an incorrect tag suggestion.